File: C to Java/Assignments/Assignment 0/q5.java
import java.util.Scanner;

public class q5 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        float a1, a2, a3, a4, a5;

        System.out.println("Enter 5 numbers:");

        a1 = sc.nextFloat();
        a2 = sc.nextFloat();
        a3 = sc.nextFloat();
        a4 = sc.nextFloat();
        a5 = sc.nextFloat();

       float avg = (a1 + a2+a3+ a4+ a5) /5 ;
	
	System.out.printf("%f" , avg)  ;
    }
}

File: C/Assignments/Assignment 0/q5.c
//5. Write a C program to input five numbers and find their average.

void main() {
	float a1 , a2, a3, a4, a5 ,avg ;
	
	printf("enter five value line by line for average \n");
	
	scanf("%f %f %f %f %f" , &a1 , &a2, &a3, &a4, &a5 ) ;
	
	avg = (a1 + a2+a3+ a4+ a5) /5 ;
	
	printf("%f" , avg)  ;
	
	
	
}
